compiler/rustc_const_eval/src/const_eval/eval_queries.rs

```diff
@@ -444,7 +425,7 @@ pub fn const_report_error<'mir, 'tcx>(
         *ecx.tcx,
         error,
         None,
-        || crate::const_eval::get_span_and_frames(ecx.tcx, &ecx.machine),
+        || crate::const_eval::get_span_and_frames(ecx.tcx, ecx.stack()),
         move |span, frames| errors::UndefinedBehavior { span, ub_note, frames, raw_bytes },
```

Comment:
`errors::UndefinedBehavior` is poorly named, this is specifically for validation failure, not all const-eval UB.